src/components/Button/src/button.module.css

```diff
@@ -0,0 +1,94 @@
+.denhaag-button {
+  font-family: var(--denhaag-font-family);
+  border-radius: var(--denhaag-border-radius);
+}
+
+.denhaag-button--focus:after, .denhaag-button:focus:after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: -2px;
+  bottom: -2px;
+  left: -2px;
+  right: -2px;
+  border: var(--denhaag-focus-border);
+  border-radius: var(--denhaag-border-radius);
+}
+
+/* Due to the accessibility focus outline overlapping with the border, the outline needs to be repositioned for the border variant */
+.denhaag-button--secondary-action.denhaag-button--focus:after, .denhaag-button--secondary-action:focus:after {
+  top: -3px;
+  bottom: -3px;
+  left: -3px;
+  right: -3px;
+}
+
+.denhaag-button--primary-action {
+  background-color: var(--denhaag-button-primary-action-background-color);
+  color: var(--denhaag-button-primary-action-color);
+}
+
+.denhaag-button--primary-action.denhaag-button--hover, .denhaag-button--primary-action:hover {
+  background-color: var(--denhaag-button-primary-action-hover-background-color);
+  color: var(--denhaag-button-primary-action-hover-color);
+}
+
+.denhaag-button--primary-action.denhaag-button--disabled {
```

Comment:
@Robbert I'm personally quite used to styling disabled with `.denhaag-button--primary-action[disabled]`. Looking at it from the perspective of reusable CSS which can be used in a default HTML project, how do you feel about adding this rule here? Is it from your perspective a bad practice to create selectors with attributes?

Reply:
And a quick follow-up question, the `disabled` attribute does this automatically so I don't think this is necessary but do we need `pointer-events: none` here?

Reply:
I agree that in most other projects I would prefer the "standards" way, so in that case I would go with the `:disabled` CSS selector which in turn looks at the `[disabled]` attribute of certain elements.

For maximum reusability, and to allow us to create Storybook stories for every state (even including `--hover`), I am currently leaning towards offering BEM modifiers for every state. The CSS selector that makes assumptions about what attributes exist could be in a separate file, as I mentioned the other day. You can have a look in this PR for the disabled state in Utrecht, where we have the `button:disabled` selector in `html.scss` whereas the BEM class names are defined in another `.css` file.

https://github.com/nl-design-system/utrecht/pull/60

`pointer-events: none` is not necessary, indeed.

A relevant interesting article about this exact subject was published a couple of days ago:
https://css-tricks.com/making-disabled-buttons-more-inclusive/